src/styles/variables.css

```diff
@@ -0,0 +1,53 @@
+:root {
+  /* ----- typography ----- */
+  --h1-font-size: 4rem;
+  --h2-font-size: 3rem;
+  --h3-font-size: 2.25rem;
+  --h4-font-size: 1.5rem;
+  --paragraph-font-size: 1.25rem;
+
+  /* ----- line-height ----- */
+  --h1-line-height: 4.75rem;
+  --h2-line-height: 3.6rem;
+  --h3-line-height: 2.7rem;
+  --h4-line-height: 1.75rem;
+  --paragraph-line-height: 1.5rem;
+
+  /* ----- colors ----- */
+  --color-orange: #f6a721;
+  --color-blue: #03628a;
+  --color-blue-light: #6daec7;
+  --color-white: #fff;
+  --color-grey: #d0d0d0;
+  --color-black: #282828;
+
+  /* ----- spacing ----- */
+  --space-xs: 4px;
+  --space-s: 8px;
+  --space-m: 16px;
+  --space-l: 24px;
+  --space-xl: 32px;
+  --space-xxl: 64px;
+  --space-xxxl: 128px;
+
+  /* ----- section ----- */
+  --section-max-width: 1120px;
+  --section-spacing-vertical: 128px;
+  --section-spacing-horizontal: 64px;
+}
+
+@media (max-width: 1024px) {
```

Comment:
We are unable to use CSS variables for breakpoints, but we can set some component/section related variables here and reference them in CSS modules - wdyt about that approach?

Reply:
Looks fine to me